packages/rs-sdk/Cargo.toml

```diff
@@ -8,19 +8,22 @@ dpp = { path = "../rs-dpp", features = [
     "client",
     "validation",
     "identity-value-conversion",
+    "state-transition-signing",
 ] }
 dapi-grpc = { path = "../dapi-grpc", features = ["client"] }
 rs-dapi-client = { path = "../rs-dapi-client", default-features = false }
 drive = { path = "../rs-drive", default-features = false, features = [
     "verify",
 ] }
 drive-proof-verifier = { path = "../rs-drive-proof-verifier" }
+dapi-grpc-macros = { path = "../rs-dapi-grpc-macros" }
 # drive-abci is only needed for core rpc; TODO remove once we have correct core rpc impl
 drive-abci = { path = "../rs-drive-abci", default-features = false }
```

Comment:
please address `cargo machete` output, https://github.com/dashpay/platform/actions/runs/7828491935/job/21358511981?pr=1596 (needs to be double-checked as machete is not perfect, for example when sth is hidden behind feature)